org.omg.kerml.xtext.ui/plugin.xml_gen

```diff
@@ -227,19 +227,6 @@
 			uriExtension="kerml">
 		</resourceServiceProvider>
 	</extension>
-	<extension
```

Comment:
Please check whether these changes need to be copied to the plugin.xml file of the same project. The basic idea here is that Xtext is not capable of updating the plugin.xml files (as it could contain manual customizations as well), so an alternative version Is generated where the developers are expected to copy the changes.

Reply:
Thanks, Zoltán. Good catch.

I compared the `plugin.xml_gen` files to the existing `plugin.xml` to confirm that all the generated changes should be included in the production files. Then I updated the production files with all the generated changes (in all three projects).